api/users.go

```diff
@@ -21,6 +21,8 @@ const (
 	sendReminderEmailPath          = "/auth/realms/:realm/onboarding/sendReminderEmail"
 	smsAPI                         = "/auth/realms/:realm/smsApi"
 	sendSmsCode                    = smsAPI + "/sendNewCode"
+	sendSmsConsentCode             = smsAPI + "/users/:userId/consent"
+	checkSmsConsentCode            = sendSmsConsentCode + "/:consentId"
```

Comment:
we call it consentCode, why do we name the parameter consentId ?

Reply:
You're right... fixed!